decidim-budgets/app/views/decidim/budgets/projects/show.html.erb

```diff
@@ -22,7 +22,7 @@ edit_link(
     <% end %>
 
     <div class="m-bottom">
-      <%= link_to resource_locator(budget).path(filter_link_params), class: "muted-link" do %>
+      <%= link_to budget_projects_path, class: "muted-link js-back-to-list" do %>
```

Comment:
This probably works fine as-is as it gets the `:budget_id` from the current params but I would rather be explicit with these, where it is linking to.

Can we pass the `budget` as an argument to the method to make it more obvious?

```ruby
budget_projects_path(budget)
```